<?xml version="1.0" ?>
<sdf version="1.6">
  <model name="qr_code_corridor">
    <static>true</static>
    <link name="link">
      <visual name="visual">
        <geometry>
          <box>
            <size>0.5 0.01 0.5</size>
          </box>
        </geometry>
        <material>
          <script>
            <uri>model://qr_code_corridor/materials/scripts</uri>
            <uri>model://qr_code_corridor/materials/textures</uri>
            
            <name>QRCode/Corridor</name> 
          </script>
        </material>
      </visual>
    </link>
  </model>
</sdf>
